[2022-09-17 14:43:31,604] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:43:29.254829+00:00 [queued]>
[2022-09-17 14:43:31,638] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:43:29.254829+00:00 [queued]>
[2022-09-17 14:43:31,638] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 14:43:31,640] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-17 14:43:31,640] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 14:43:31,753] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): t1> on 2022-09-17 14:43:29.254829+00:00
[2022-09-17 14:43:31,771] {standard_task_runner.py:52} INFO - Started process 1641 to run task
[2022-09-17 14:43:31,775] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xcom_dag', 't1', 'manual__2022-09-17T14:43:29.254829+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpnpwf4j80', '--error-file', '/tmp/tmp6oeq1ldn']
[2022-09-17 14:43:31,779] {standard_task_runner.py:80} INFO - Job 17: Subtask t1
[2022-09-17 14:43:31,975] {task_command.py:369} INFO - Running <TaskInstance: xcom_dag.t1 manual__2022-09-17T14:43:29.254829+00:00 [running]> on host 3f1973d04161
[2022-09-17 14:43:32,199] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T14:43:29.254829+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-17T14:43:29.254829+00:00
[2022-09-17 14:43:32,262] {python.py:173} INFO - Done. Returned value was: None
[2022-09-17 14:43:32,296] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=t1, execution_date=20220917T144329, start_date=20220917T144331, end_date=20220917T144332
[2022-09-17 14:43:32,432] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-17 14:43:32,553] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
